### PR TITLE
Adding Description to the arguments and fields

### PR DESCRIPTION
**Pull request by brcase-ut**
## Description, motivation and context
Adding Description to the arguments and fields

Some CI tools are set to require descriptions on fields and arguments

## Related issue(s) or PR(s)
https://github.com/RenoFi/graphql-pagination/issues/148

### CODE REVIEW

**Comment by knapo** [comment]
Thanks!

**Review by knapo — APPROVED** [review]

File: lib/graphql_pagination/collection_type.rb [hunk 1]
```diff
@@ -14,8 +14,8 @@ def collection_type(
 
         Class.new(collection_base) do
           graphql_name type_name
-          field :collection, [source_type], null: false
-          field :metadata, metadata_type, null: false
+          field :collection, [source_type], null: false, describe: "A collection of paginated #{graphql_name}"
+          field :metadata, metadata_type, null: false, describe: "Pagination Metadata for navigating the Pagination"
```
Comment (knapo) [inline lib/graphql_pagination/collection_type.rb:18]: raises:

```
ArgumentError:
  unknown keyword: :describe
```

**Review by knapo — CHANGES_REQUESTED** [review]

File: lib/graphql_pagination/collection_type.rb (hunk 1 above)
Comment (brcase-ut) [reply lib/graphql_pagination/collection_type.rb]: Sorry about that, i updated it to be `description`

**Review by knapo — APPROVED** [review]

**Comment by dpena-ut** [comment]
@knapo Thanks for the review. I see you added the `automerge` label and I found your workflow that is supposed to merge it. Looks like that failed for some reason although it isn't clear to me why.

```
2023-09-15T06:02:11.048Z INFO  PR is probably ready: mergeable_state: unstable
2023-09-15T06:02:11.328Z INFO  Failed to merge PR: Resource not accessible by integration
2023-09-15T06:02:11.328Z INFO  Retrying after 5000 ms ... (1/6)
2023-09-15T06:02:17.117Z INFO  Failed to merge PR: Resource not accessible by integration
2023-09-15T06:02:17.117Z INFO  Retrying after 5000 ms ... (2/6)
2023-09-15T06:02:22.797Z INFO  Failed to merge PR: Resource not accessible by integration
2023-09-15T06:02:22.797Z INFO  Retrying after 5000 ms ... (3/6)
2023-09-15T06:02:28.478Z INFO  Failed to merge PR: Resource not accessible by integration
2023-09-15T06:02:28.479Z INFO  Retrying after 5000 ms ... (4/6)
2023-09-15T06:02:34.161Z INFO  Failed to merge PR: Resource not accessible by integration
2023-09-15T06:02:34.161Z INFO  Retrying after 5000 ms ... (5/6)
2023-09-15T06:02:39.886Z INFO  Failed to merge PR: Resource not accessible by integration
2023-09-15T06:02:39.886Z INFO  Retrying after 5000 ms ... (6/6)
2023-09-15T06:02:45.592Z INFO  Failed to merge PR: Resource not accessible by integration
2023-09-15T06:02:45.592Z INFO  PR not ready to be merged after 6 tries
2023-09-15T06:02:45.592Z INFO  Action result: { mergeResult: 'merge_failed', pullRequestNumber: 149 }
```

**Comment by dpena-ut** [comment]
The workflow seems to have a problem with other folks as well: https://github.com/pascalgn/automerge-action/issues/236 Thanks for merging manually.

**Comment by knapo** [comment]
Thanks, I'll look into automerge issue later.

**Pull request merged**